src/styles/typing.css

```diff
@@ -0,0 +1,78 @@
+.sc-typing {
+  padding-top: 20px;
+  width: 300px;
+  margin: auto;
+}
+
+.sc-typing--circle {
+  width: 28px;
+  height: 22px;
+  padding: 5px 8px;
+  border-radius: 15px;
+  background-color: #e9ebee;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
```

Comment:
I don't think we need all these vendor prefixes, do we? If we really do then we should make another ticket to add autoprefixer.